features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/RumFeature.kt

```diff
@@ -186,14 +186,25 @@ internal class RumFeature(
         }
         configuration.longTaskTrackingStrategy?.let { longTaskTrackingStrategy = it }
 
-        initializeVitalMonitors(configuration.vitalsMonitorUpdateFrequency)
+        val frequency = configuration.vitalsMonitorUpdateFrequency
+        val slowFrameListenerConfiguration = configuration.slowFramesConfiguration
+        if (frequency != VitalsUpdateFrequency.NEVER || slowFrameListenerConfiguration != null) {
+            initializeVitalExecutorService(frequency)
+            initializeCpuVitalMonitor(frequency)
+            initializeMemoryVitalMonitor(frequency)
+            initializeFrameStatesAggregator(
+                application = appContext as? Application,
+                listeners = listOfNotNull(
+                    initializeSlowFrameListener(slowFrameListenerConfiguration),
```

Comment:
side question: Will Slow frames reporting be a part of "Mobile Vitals" umbrella? Because right now we have kind of 2 public APIs to monitor the app health: vitals update frequency and slow frames reporting, and disabling the former doesn't disable the latter, which may sometimes bring a confusion.

Reply:
we gonna discuss it with ios team